src/lib/moment/start-end-of.js

```diff
@@ -1,59 +1,127 @@
 import { normalizeUnits } from '../units/aliases';
+import { hooks } from '../utils/hooks';
+
+var MS_PER_SECOND = 1000;
+var MS_PER_MINUTE = 60 * MS_PER_SECOND;
+var MS_PER_HOUR = 60 * MS_PER_MINUTE;
+var MS_PER_DAY = 24 * MS_PER_HOUR;
```

Comment:
I don't mind renaming this back to 24_HR_DAY or what you had before, especially if you leave a comment that some days during DST do not have 24 hours.

Reply:
As the MS_PER_DAY variable's only used once, I've inlined to avoid any further debate :-)